forms-flow-web/src/components/ServiceFlow/details/ServiceTaskDetails.js

```diff
@@ -242,7 +242,8 @@ const ServiceFlowTaskDetails = React.memo(() => {
 
   const onFormSubmitCallback = (actionType = "", successPage) => {
     if (bpmTaskId) {
-      dispatch(setBPMTaskDetailLoader(true));
+      // comment out the following line. It is showing the 3 dots animation.
+      // dispatch(setBPMTaskDetailLoader(true));
```

Comment:
@jerry-ey 
If we disable the default loaded, does it mean all the forms should send an event to trigger the loader onSubmission?

Can you please test other forms with this change to see their current behaviour doesn't break? I belive only Telework form send event for loader.

Reply:
Good call, I will double check and make necessary changes.